source/boot_security.c

```diff
@@ -1,11 +1,21 @@
 #include "boot_security.h"
```

Comment:
Guys, you are doing really well so could you please split the code using some comments. 

Please see the link for details. 

https://github.com/OpenIoTAcademy/psa-boot-beta/wiki/File-Comments-in-Doxygen-Style

Let us collect related parts in the same section in the file.